apps/condo/pages/tls.tsx

```diff
@@ -0,0 +1,296 @@
+// @ts-ignore
+import enMacos from '@condo/lang/en/pages/tls/macos.md'
+// @ts-ignore
+import enWindows from '@condo/lang/en/pages/tls/windows.md'
+// @ts-ignore
+import ruMacos from '@condo/lang/ru/pages/tls/macos.md'
+// @ts-ignore
+import ruWindows from '@condo/lang/ru/pages/tls/windows.md'
+import styled from '@emotion/styled'
+import { Anchor, Col, Collapse, Image, Row } from 'antd'
+import get from 'lodash/get'
+import Link from 'next/link'
+import { useRouter } from 'next/router'
+import React, { CSSProperties, useCallback } from 'react'
+
+import { useAuth } from '@open-condo/next/auth'
+import { useIntl } from '@open-condo/next/intl'
+import { Markdown, Typography } from '@open-condo/ui'
+import { Button, Card, Space } from '@open-condo/ui'
+import { colors } from '@open-condo/ui/dist/colors'
+
+import EmptyLayout from '@condo/domains/common/components/containers/EmptyLayout'
+import { useLayoutContext } from '@condo/domains/common/components/LayoutContext'
+import { Logo } from '@condo/domains/common/components/Logo'
+import { Poster } from '@condo/domains/common/components/Poster'
+import { antGlobalVariables, fontSizes } from '@condo/domains/common/constants/style'
+import { PosterWrapper } from '@condo/domains/user/components/containers/styles'
+import { QuestionCircle } from '@open-condo/icons'
+
+const LOGO_HEADER_STYLES = { width: '100%', justifyContent: 'space-between' }
+const HEADER_LOGO_STYLE: React.CSSProperties = { cursor: 'pointer' }
+
+const guidesContent = {
+    ru: {
+        windows: ruWindows,
+        macos: ruMacos,
+    },
+    en: {
+        windows: enWindows,
+        macos: enMacos,
+    },
+}
+
+export const Header = () => {
+    const intl = useIntl()
+    const BackMessage = intl.formatMessage( { id: 'pages.condo.tls.back' })
+
+    const { push } = useRouter()
+    const { isAuthenticated } = useAuth()
+
+    const handleLogoClick = useCallback(() => {
+        if (isAuthenticated) {
+            push('/')
+        } else {
+            push('/auth/signin')
+        }
+    }, [isAuthenticated, push])
```

Comment:
For disable warning messages it would better to change this handler to that one:
```suggestion
    const handleLogoClick = useCallback(async () => {
        if (isAuthenticated) {
            await push('/')
        } else {
            await push('/auth/signin')
        }
    }, [isAuthenticated, push])
```